docs/CustomModelInstructions.md

```diff
@@ -0,0 +1,29 @@
+# Custom Model Training Instructions
+
+Follow these steps to train a custom model using the YOLOv8 algorithm.
+
+## Step 1: Unzip Training Data
```

Comment:
maybe mention that one could input their own training data instead of the one we provide?

Reply:
Yeah true, that would make sense. Maybe we could even provide an interface in streamlit, so that you would only need to drag & drop your file? Or would that be unnecessary?

Reply:
Hmm not sure about that one. At least not for now.. Given the amount of files we would have to upload and save its easier to simply provide a manual way to accomplish that. 

Like we have other demands for now than to implement that ^

Reply:
okok, will only update the instructions

Reply:
Thanks